### DISCUSSION TITLE
Github EdGithub Education Pack is not approved.ucation

### DISCUSSION THREAD

**Discussion by pioneerTG** [community/community#53423] [GitHub Education] [answered]
Github Education Pack is not approved.
I'm currently signed up with a school verified email ending in ac.kr, and when I upload my student ID and wait for approval, I get rejected every time.
I don't know what could be the problem.

**Comment by pioneerTG** [comment]
The comment is shown below. ▽

PronounsHey pioneerTG,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Apr 21, 2023 for @pioneerTG.

You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.

Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.


Once you have resolved the issues with your sumbission, visit us again and reapply.

Have questions? Join the GitHub Education Community for answers.

💖 The GitHub Education Team

**Comment by juanpflores** [comment, marked as answer]
Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍
